WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeListActivity.java

```diff
@@ -249,6 +259,30 @@ public void onRequestDisconnect(PublicizeConnection publicizeConnection) {
         confirmDisconnect(publicizeConnection);
     }
 
+    private boolean isFacebook(PublicizeService service) {
+        return service.getId().equals(PublicizeConstants.FACEBOOK_ID);
+    }
+
+    private String getConnectionsUrl(SiteModel site) {
+        return "https://wordpress.com/marketing/connections/" + SiteUtils.getHomeURLOrHostName(site);
```

Comment:
Should this URL go into constants?  Is it because it is a temporary fix!

Reply:
Good catch! It wasn't intentional, I guess I was just being lazy 😅

Fixed in a96f815c255a2d8f9ba1c05569df0820f467686d.